sql/2020/07_SEO/lighthouse.sql

```diff
@@ -41,8 +74,10 @@ FROM (
     JSON_EXTRACT_SCALAR(report, '$.audits.meta-description.score') = '1' AS has_meta_description,
     JSON_EXTRACT_SCALAR(report, '$.audits.image-alt.score') = '1' AS img_alt_on_all,
     JSON_EXTRACT_SCALAR(report, '$.audits.robots-txt.score') = '1' AS robots_txt_valid,
-    JSON_EXTRACT_SCALAR(report, '$.audits.link-text.score') = '1' AS link_text_descriptive
+    JSON_EXTRACT_SCALAR(report, '$.audits.link-text.score') = '1' AS link_text_descriptive,
+    isCrawlable(report) AS isCrawlable
```

Comment:
Use underscores for consistency. Also as per above can we distinguish the `is_crawlable` colmns?

```suggestion
    isCrawlable(report) AS is_crawlable_details
```